lib/processor.js

```diff
@@ -12,6 +12,9 @@ var assign = require("object-assign");
 var remark = require("remark");
 
 var SUPPORTED_SYNTAXES = ["js", "javascript", "node", "jsx"];
+var UNSATISFIABLE_RULES = [
```

Comment:
`settings` only gets passed to rules though. If necessary, we could:

1. Modify core to pass `settings` to processors.
2. Hack it in by offering a `markdown/unsatisfiable-rules` meta-rule that would remember its config.
3. Shave an entire herd of yacks and get glob config in core.